backpressure/src/main/java/com/iluwatar/backpressure/Subscriber.java

```diff
@@ -0,0 +1,43 @@
+package com.iluwatar.backpressure;
+
+import lombok.NonNull;
+import lombok.extern.slf4j.Slf4j;
+import org.reactivestreams.Subscription;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import reactor.core.publisher.BaseSubscriber;
+
+/** This class is the custom subscriber that subscribes to the data stream. */
+@Slf4j
+public class Subscriber extends BaseSubscriber<Integer> {
+
+  private static final Logger logger = LoggerFactory.getLogger(Subscriber.class);
```

Comment:
No need to declare logger like this. With the Lombok annotation you automatically have LOGGER available.